### ISSUE TITLE
Errno::ENOENT: No such file or directory @ rb_sysopen

### ISSUE THREAD

**Issue by KevinColemanInc** [Compass/compass#1918]
I think this is you guys

```
Errno::ENOENT: No such file or directory @ rb_sysopen - /Users/kcoleman/.rvm/gems/ruby-2.1.5/gems/compass-1.0.2/lib/compass/../../VERSION_NAME
```

**Comment by tamird** [comment]
:+1: also getting this in ruby 2.2

**Comment by tamird** [comment]
dupe of #1917

**Comment by KevinColemanInc** [comment]
this completely killed my app, because rails-compass doesn't lock the version...

**Comment by pi-ron** [comment]
I'm getting this error after update a projects gemfile, `bundle install` ran fine, `compass compile --trace`throws list of errors, with this at the start.

`ruby -v 2.0.0`

**Comment by pi-ron** [comment]
Changing my bundle to use compass '1.0.1' (instead of 1.0.2), fixed the issue for me.

**Comment by KevinColemanInc** [comment]
@Tigeda, yes, but if you are using compass-rails like I was, you have to edit your Gem lock file as well :-1:

**Comment by scottdavis** [comment]
dupe of #1917

**Comment by scottdavis** [comment]
This should be fixed by d5df161 please update and let me know if it works
